app/uk/gov/hmrc/cataloguefrontend/DependencyReportController.scala

```diff
@@ -107,8 +107,8 @@ class DependencyReportController @Inject()(
       case Some(VersionState.MinorVersionOutOfDate) => "amber"
       case Some(VersionState.MajorVersionOutOfDate) => "red"
       case Some(VersionState.Invalid)               => "N/A"
-      case Some(VersionState.BobbyRuleViolated)     => "verybad"
-      case Some(VersionState.BobbyRulePending)      => "pending"
+      case Some(_: VersionState.BobbyRuleViolated)  => "verybad"
+      case Some(_: VersionState.BobbyRulePending)   => "pending"
```

Comment:
Minor but wondering if we should rename this to `BobbyRuleWarning` to match `sbt-bobby`? Also sounds a bit like it's waiting to be create or something